tests/cross/src/test/scala/tests/pc/AutoImplementAbstractMembersSuite.scala

```diff
@@ -1277,7 +1277,6 @@ class AutoImplementAbstractMembersSuite extends BaseCodeActionSuite {
        |
        |trait Base:
        |  def foo(x: Int): Int
-       |  def bar(x: String): String
```

Comment:
Why remove these?

Reply:
Because the ordering doesn't seem to be stable (or somehow this PR changed it for `3.3.1`, though it really doesn't seem related). I can probably sort but for this example it doesn't matter if there is more than one method.

Reply:
it would be good though to have them implemented the same way they are in the parent trait. I can imagine people complaining if this is not the case